player_rating/tests.py

```diff
@@ -2,38 +2,12 @@
 from player.models import BallGame, Player
 from django.core.exceptions import ValidationError
 from django.db import IntegrityError
-from django.contrib.auth import get_user_model
 import pytest
 
 
 @pytest.mark.django_db
 class TestPlayerModel:
 
-    @pytest.fixture
-    def user(self):
-        return get_user_model().objects.create_user(
-            username='testuser',
-            password='testpass'
-        )
-
-    @pytest.fixture
-    def player(self, user):
-        player = Player.objects.create(
-            user=user,
-            birth_date='1990-01-01',
-            favorite_ball_game=BallGame.Soccer
-        )
-        return player
-
-    @pytest.fixture
-    def player_rating(self, player):
-        player_rating_fixture = PlayerRating.objects.create(
-            ball_game=BallGame.Soccer,
-            user_id=player,
-            rating=Rating.Five
-        )
-        return player_rating_fixture
```

Comment:
Are you expecting to use this fixture anywhere but within this file? If not, then I don't see a reason to have it moved.